import org.apache.hadoop.fs.PathIOException;
public class FunctionUse13053 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
}
}